openwpm/deploy_browsers/deploy_firefox.py

```diff
@@ -110,16 +117,30 @@ def deploy_firefox(
         # TODO restore detailed logging
         # fo.set_preference("extensions.@openwpm.sdk.console.logLevel", "all")
 
+    # Geckodriver currently places the user.js file in the wrong profile
+    # directory, so we have to create it manually here.
+    # TODO: Remove this workaround once
```

Comment:
File a new issue for this removal and reference it here. (We don't want to track TODOs in the codebase).